docs/reference/context.md

```diff
@@ -59,7 +59,7 @@ context is known. The body of a context expression may be evaluated zero, one,
 or multiple times, depending on how many different places it is put into.
 
 ## Location context
-Context can not only give us access to set rule values. It can also let us know
+Context can give us not only access to set rule values, but also lets us know
```

Comment:
That form suggests that context also gives us "lets us know where in the document we currently are", which isn't what is being said. In the end, the original phrase was mostly correct, but needed a comma with a connective instead of a period to feel more natural.

```suggestion
Context can not only give us not only access to set rule values, but also let us know
```

Reply:
@PgBiel your suggested change contains two times "not" now. But I agree that just changing period to comma in the original sentence would be good.

Reply:
Thanks, it was a typo.

Reply:
Fair.

Reply:
Although, the "can" and "let" are so far apart that perhaps isn't not the best way, so I kinda half-skipped the suggestion and changed the phrase to:

> Not only context can give us access to set rule values, but also lets us know

Reply:
Sorry, but your new version doesn't read naturally to me. You'd have to write "not only can context" instead. And still, it's weird as you aren't specifying something else that it can do later, but rather changing the verb. So I believe my original suggestion is the easiest path.

Reply:
I'd suggest just going really easy, especialy with non-native speakers in mind: "Context gives us access to set rules values, and lets us know...".

Reply:
@KillTheMule I took the more defensive phrasing because in rare cases context can also be partial and only provide one of the things, so "context gives us access..." isn't necessarily correct.

Reply:
Are there any examples of partial context? I don't remember reading this in docs, but I think these kinds of details are important, so that the user wouldn't fall in a trap.

Reply:
When location context is available, style context is also always available currently (though this is not an invariant at the compiler level). But the opposite is not necessarily true. For example, table cell properties are resolved with styles, but without a location.

So you can write `table(fill: (..) => text.fill)`, but not `table(fill: (..) => { here(); red })`.